configs/encapsulate_in_http1_connect.yaml

```diff
@@ -27,6 +27,25 @@ static_resources:
           cluster: "cluster_0"
           tunneling_config:
             hostname: host.com:443
+  - name: listener_1
```

Comment:
I'm not sure what these config changes are demonstrating. Can you document these somewhere and/or add inline comments to make it more clear what people should be learning from this? Same elsewhere.

Reply:
I add the part that this listener takes the tcp data from port 10002 to upstream CONNECT, adding the "foo:bar" in the CONNECT request.

And the curl command to examine